analytics.go

```diff
@@ -283,13 +302,13 @@ func (c *Client) loop() {
 			msgs = append(msgs, msg)
 			if len(msgs) == c.Size {
 				c.verbose("exceeded %d messages – flushing", c.Size)
-				c.send(msgs)
+				go c.send(msgs)
```

Comment:
@tj Very rough draft of what a bounded executor could look like - https://github.com/segmentio/analytics-go/commit/db3adc6760502971fa7356219fd2ba857900f20e